CONTRIBUTING.md

```diff
@@ -335,6 +335,31 @@ since the latest release over the hotfixed `master`.
 This would mean, however, that the commits referenced from PRs previously merged to `develop` will no longer be part of `develop`'s history,
 which is rather unacceptable.
 
+### Plugin signing
+
+The valid non-expired key pair together with the password required for the plugin signing should be present in the CI environment.
+If they are absent, please take a look at the [plugin signing in IntelliJ](https://plugins.jetbrains.com/docs/intellij/plugin-signing.html#signing-methods) for updated instructions on how to do it.
+Currently, you would need to first generate the private key with:
+```shell
+openssl genpkey\
+  -aes-256-cbc\
+  -algorithm RSA\
+  -out private.pem\
+  -pkeyopt rsa_keygen_bits:4096\
+  -pass env:PASSWORD_ENVIRONMENT_VARIABLE_NAME
+```
+Then, use that for generating the certificate chain by running:
+```shell
+openssl req\
+  -key private.pem\
+  -new\
+  -x509\
+  -days 365\
+  -out chain.crt\
+  -passin env:PASSWORD_ENVIRONMENT_VARIABLE_NAME\
```

Comment:
Let's just use the actual env var name as expected by Gradle, not a placeholder name

Reply:
I'm afraid, this is an env var on the developer's local machine, so it is on the developer to create this env variable. Do you mean it would be best to use the same env var name we have on the CI?

Reply:
Yeah, let's be consistent